docs/gc.md

```diff
@@ -130,17 +130,47 @@ There are a number of advantages to using arraylets.
+
+Overall, these policies are best suited to short-lived applications and to long-running services involving concurrent sessions that have short lifespans. Short-lived applications with adequate heap sizes usually complete without compaction, and the flat heap fragments more slowly when session-bound objects are allocated and drop out of the live set in short overlapping clusters.
+
+#### `optavgpause` policy
+
+[`-Xgcpolicy:optavgpause`](xgcpolicy.md#optavgpause) uses concurrent marking, as for `gencon`, to anticipate global collections and initiate some marking before the stop-the-world global collection phase. This reduces the likelihood of long interruptions of service due to GC activity and pause times are reduced when compared to `optthruput` though at the expense of some performance throughput.
+
+#### `optthruput` policy
+
+[`-Xgcpolicy:optthruput`](xgcpolicy.md#optthruput) is optimized for throughput by disabling the concurrent mark phase, which means that applications stop for long pauses while garbage collection takes place. You might consider using this policy when your application can tolerate longer GC pauses to obtain better overall throughput.
+
+
 ### `metronome` policy
 
-[`-Xgcpolicy:metronome`](xgcpolicy.md#metronome-aix-linux-x86-only) is designed for applications that require precise response times. Garbage collection occurs in small interruptible steps to avoid stop-the-world pauses. This policy is available only on x86 Linux and AIX platforms.
+**(Linux on x86-64 and AIX platforms only)**
+
+[`-Xgcpolicy:metronome`](xgcpolicy.md#metronome-aix-linux-x86-only) is designed for applications that require a precise upper bound on collection pause times as well as specified application utilization: the proportion of time that the application is permitted to use, with the remainder being devoted to GC. The realtime GC runs in short interruptible bursts to avoid long stop-the-world pauses.
+
+The realtime heap is allocated as a contiguous range of RAM partitioned into small regions of equal size, usually about 64 KB. There is no dynamic heap resizing when you use the `metronome` policy. The heap is always fully expanded even if [`-Xms`](xms.md) is not set to [`-Xmx`](xms.md).
+
+As with the `balanced` GC policy, arrays are represented as arraylets with a spine pointing to a series of regions containing the array elements (see [Arraylets: dealing with large arrays](#arraylets-dealing-with-large-arrays) for more information). Note that JNI access to array data might involve reconstituting arraylets as contiguous arrays, which can significantly slow down processing.
+
+Each region of the heap is either empty, or contains only objects in one of 16 size classes, or an arraylet. This organization reduces heap fragmentation, so that need for heap compaction and defragmentation is minimized, and provides more precise control of the incremental sweep phase.
```

Comment:
Had trouble reading this .... maybe we need to add a "the" ... " so that the need" ... and of/over to "more precise control over the incremental *sweep* operation". Hmm ... still struggling with ", and provides". 

Maybe:

"This organization improves the use of available heap space, reducing the need for heap compaction and defragmentation, and providing more precise control over the incremental *sweep* operation."

Note: We are calling `<gc-op>`s "operations" with the name in *italics*. Probably easier to sweep the whole topic once we have it merged.

Reply:
Reworded as suggested.